.prow.yaml

```diff
@@ -0,0 +1,45 @@
+presubmits:
+  - name: pull-apimachinery-verify
+    always_run: true
+    decorate: true
+    clone_uri: "ssh://git@github.com/kcp-dev/apimachinery.git"
+    labels:
+      preset-goproxy: "true"
+    spec:
+      containers:
+        # TODO(nabarun): Change the following to the kcp-dev/build image once it is available
+        # Ref: https://github.com/kcp-dev/infra/pull/34
+        - image: quay.io/kubermatic/build:go-1.20-node-18-6
+          command:
+            - make
+            - verify-boilerplate
+
+  - name: pull-apimachinery-lint
+    always_run: true
+    decorate: true
+    clone_uri: "ssh://git@github.com/kcp-dev/apimachinery.git"
+    labels:
+      preset-goproxy: "true"
+    spec:
+      containers:
+        # TODO(nabarun): Change the following to the kcp-dev/build image once it is available
+        # Ref: https://github.com/kcp-dev/infra/pull/34
+        - image: quay.io/kubermatic/build:go-1.20-node-18-6
```

Comment:
Please update to `ghcr.io/kcp-dev/infra/build:1.19.9-2` now :)

Reply:
The PR is updated with the latest commits now. 🎉